javalib/src/main/scala/java/util/TimerTask.scala

```diff
@@ -0,0 +1,30 @@
+package java.util
+
+import scala.scalajs.js.timers
+
+abstract class TimerTask {
+  private[util] var cancelled: Boolean = false
+  private[util] var lastScheduled: Long = 0L
+
+  def run(): Unit
+
+  def cancel(): Unit = {
+    cancelled = true
+  }
+
+  def scheduledExecutionTime(): Long = lastScheduled
+
+  private[util] def doRun() {
+    if (!cancelled) {
+      lastScheduled = System.currentTimeMillis()
+      try {
+        run()
+      } catch {
+        case t: Throwable =>
+          cancelled = true
+          throw t
```

Comment:
As far as I know, an exception cancels the timer task, but this is not specified in the docs (e.g. http://stackoverflow.com/questions/8743027/java-timer-class-timer-tasks-stop-to-execute-if-in-one-of-the-tasks-exception-i).